sigar_windows_test.go

```diff
@@ -95,4 +96,78 @@ var _ = Describe("SigarWindows", func() {
 			Ω(sd.Description).Should(Equal("Windows"))
 		})
 	})
+
+	Describe("ProcessList", func() {
+		It("gets the list of processes", func() {
+			pl := sigar.ProcList{}
+			err := pl.Get()
+			Ω(err).ShouldNot(HaveOccurred())
+			Ω(len(pl.List)).Should(BeNumerically(">", 0))
+		})
+	})
+
+	Describe("ProcessState", func() {
+		It("gets the process detail", func() {
+			pl := sigar.ProcList{}
+			err := pl.Get()	
+			Ω(err).ShouldNot(HaveOccurred())
+			for _, pid := range pl.List {
+				details := sigar.ProcState{}
+				err  := details.Get(pid)
+				fmt.Printf("Pid: %v Name: %v Err: %v\n", pid, details.Name, err)
```

Comment:
Yeah, the printed output isn't too useful. Using a different command as an oracle is possible, but it introduces a lot of room for flakiness. Long-term I'd like to mock all of the API calls, but I think it would be a pretty time-consuming endeavour.